File: src/server/TypeORM/src/entities/agile_story.ts
import {Entity, PrimaryGeneratedColumn, Column} from "typeorm";

@Entity()
export class agile_story {

    @PrimaryGeneratedColumn()
    agile_story_id: string;

    @Column()
    agile_story_name: string;

    @Column()
    agile_sprint_id: string;

    @Column()
    story_description: string;

    @Column()
    story_type: string;

    @Column()
    story_status: string;

    @Column()
    story_points: string;

}

File: fg_metrics/VSTS/VSTSComms/VSTSComms/EntitiesMySQL/AgileStory.cs
using System;
using System.Collections.Generic;

namespace VSTSComms.EntitiesMySQL
{
    public partial class AgileStory
    {
        public AgileStory()
        {
            AgileStoryAgileSystemUser = new HashSet<AgileStoryAgileSystemUser>();
        }

        public string AgileStoryId { get; set; }
        public string AgileStoryName { get; set; }
        public string AgileSprintId { get; set; }
        public string StoryDescription { get; set; }
        public string StoryType { get; set; }
        public string StoryStatus { get; set; }
        public string StoryPoints { get; set; }

        public AgileSprint AgileSprint { get; set; }
        public ICollection<AgileStoryAgileSystemUser> AgileStoryAgileSystemUser { get; set; }
    }
}
